src/content/developers/docs/consensus-mechanisms/pos/block-proposal/index.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Block proposal
+description: Explanation of how blocks are proposed in proof-of-stake Ethereum.
+lang: en
+---
+
+Blocks are the fundamental units of the blockchain. Blocks are discrete units of information that get passed between nodes, agreed upon and added to each node's database. This page explains how they are produced.
+
+## Prerequisites {#prerequisites}
+
+Block proposal is part of the proof-of-stake protocol. To help understand this page, we recommend you read about [proof-of-stake](src/content/developers/docs/consensus-mechanisms/pos/) and [block architecture](src/content/developers/docs/blocks/).
+
+## Who produces blocks? {#who-produces-blocks}
+
+Blocks are proposed by validator accounts. Validator accounts are managed by node operators who run validator software as part of their execution and consensus clients, and have deposited at least 32 ETH into the deposit contract. However, each individual validator is only occasionally responsible for proposing a block. Ethereum measures time in slots and epochs. Each slot is twelve seconds, and 32 slots (6.4 minutes) make up an epoch. Every slot is an opportunity to add a new block on Ethereum.
+
```

Comment:
A H3 here might break this up a little for readability?

```suggestion

### Random selection {#random-selection}

```

I'm not sold on my choice of heading name if you have an alternative suggestion 😆

Reply:
i think that's ok